lib/nice_maps.ex

```diff
@@ -224,4 +224,76 @@ defmodule NiceMaps do
     key_type = Keyword.get(opts, :key_type)
     parse_key_type(key, key_type)
   end
+
+  @doc """
+  Merges the values of two given maps.
+
+  ## Options
+
+    - `:keys` (optional) only merge the given keys
+    - `:fun`  (optional) merge handler annonymous function that accepts two arguments
+
+  ## Examples
+
+      iex> acc_requests = %{success: ["200", "200", "201"], failed: []}
+      iex> new_requests = %{success: ["200", "200"], failed: ["404"]}
+      iex> NiceMaps.merge_values(acc_requests, new_requests)
+      %{success: ["200", "200", "201", "200", "200"], failed: ["404"]}
+
+      iex> acc_requests = %{success: ["200", "200", "201"], failed: []}
+      iex> new_requests = %{success: ["200", "200"], failed: ["404"]}
+      iex> NiceMaps.merge_values(acc_requests, new_requests, keys: [:success])
+      %{success: ["200", "200", "201", "200", "200"]}
```

Comment:
Needs a few more examples for the other options ^^

Reply:
Added a few more.